### ISSUE TITLE
feat: add feature/setting to pick pre-release patches instead 

### ISSUE THREAD

**Issue by 0frd** [ReVanced/revanced-manager#1867]
### Feature description

Kindly add feature/setting to switch using  pre-release patches instead the release one. Sometime some fixes patches already exist but its on pre-release and take to long until the patches release, and also really helpful to test new patches working or not. I know we can using cli, but not everybody have pc. 
![Screenshot_2024-04-14-19-55-23-307_app rvx manager flutter-edit](https://github.com/ReVanced/revanced-manager/assets/72485140/4eda45b6-57ca-47a3-a0e0-eabb15c441fb)

### Motivation

Yo improve user experience

### Acknowledgements

- [X] This issue is not a duplicate of an existing feature request.
- [X] I have chosen an appropriate title.
- [X] The feature request is only related to ReVanced Manager

**Comment by oSumAtrIX** [comment]
No, but this feature is being looked into as we speak coincidentally. API already supports it, manager just needs to add the toggle

**Comment by Keith94** [comment]
PR just opened here: https://github.com/ReVanced/revanced-manager/pull/2485
